src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexNode.cs

```diff
@@ -1360,14 +1360,15 @@ static void ProcessOneOrMulti(RegexNode node, ReadOnlySpan<char> startingSpan)
         /// </summary>
         public RegexNode? FindBranchOneOrMultiStart()
         {
-            RegexNode branch = this;
-
-            if (branch.Type == Concatenate)
-            {
-                branch = branch.Child(0);
-            }
+            RegexNode branch = Type == Concatenate ? Child(0) : this;
```

Comment:
Nit, I think we usually wrap so it's easier to read?
```suggestion
            RegexNode branch = (Type == Concatenate) ? Child(0) : this;
```
and below.